src/frontend.cpp

```diff
@@ -1,10 +1,18 @@
 #include "frontend.h"
-#include <ostream>
 
 void Frontend::GenerateStringStream() {
 	stream.str(drv->out);
 }
 
+bool Frontend::ValidateFileStream(const std::string &filename) {
+	std::ifstream fileStream(filename.c_str());
+	if (!fileStream.good()) {
```

Comment:
If this function is used to check if an argument is a file, but it fails, because it's actually `-o`, it will still print the Exception? That seems like a bad idea.